code/modules/research/nanites/nanite_programs/healing.dm

```diff
@@ -176,6 +180,16 @@
 				update = TRUE
 		if(update)
 			host_mob.update_damage_overlays()
+		if(C.getStaminaLoss() < 100) //Stops after hitting the second slowdown level. This is distributed so only ~80% is counted towards crit
+			C.adjustStaminaLoss(5) //Hurts a lot more
+		else if(C.getBruteLoss() || C.getFireLoss()) //Prevents stamina regen if it's actively healing and you're over the limit.
+			C.adjustStaminaLoss(0.1) //Not healing any wounds, so the nanites will reconstruct random healthy tissue. Very painful!
```

Comment:
Im not sure what you mean by this comment as this code triggers only when healing wounds

Reply:
whoops, put the comment in the wrong spot